test/values.ts

```diff
@@ -674,6 +674,19 @@ export const mockProviders: FetchProvidersOutput[] = [
     cashIn: true,
     cashOut: false,
   },
+  {
+    name: 'Ramp',
+    restricted: false,
+    unavailable: false,
+    paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
```

Comment:
nit: will only be Bank for cash outs

Reply:
yeah, but just wanted to verify that we do in fact generate one quote per payment method in this case.